support/helper.py

```diff
@@ -11,6 +11,7 @@ def head_tail_list(lst):
 
 
 def aggregate_currencies(ccy1, ccy2, fiats):
+    ccy1 = {k:v[1] for (k, v) in ccy1.items()}
```

Comment:
@irene-bbox what are the impacts on coinbase model? same as the comments on [https://github.com/BalloonBox-Inc/NEARoracle-Oracle/pull/34/files#diff-ac46890f94fd4daea75b9542f384662762d11cb51efaee585c257eafa2fa294c](url)

Reply:
Thank you @m3mayllon ! Resolved. See [here](https://github.com/BalloonBox-Inc/NEARoracle-Oracle/pull/34/files#r913225865).